### ISSUE TITLE
The selection of keyboards in the taskbar disappears.

### ISSUE THREAD

**Issue by zbigniew59** [undergroundwires/privacy.sexy#24]
This scrypt removes my choice of taskbar keyboards. I use 2 different keyboards and now I have no choice at all.
What is responsible for this?

**Comment by undergroundwires** [comment]
Hi @zbigniew59 , thanks for the bug report. This is unintended. Let's see what caused it & how we can revert it. I'll need more information to be able to take a closer look into this:

- Can you copy & paste or attach the generated script file that you executed?
- Which version of Windows are you running ? *(Start menu -> enter About your PC -> Post what you see as "version" under "Windows Specifications")*

**Comment by mdl052020** [comment]
@Disk2019 Welcome Again.

Actually what is your POV according to your Ques.
i cant even undertand your query .
is it related to in build software keyboard or a physical hardware keyboard.
or
you are not able to choose input languages in keyboards either physical or in build.

**Comment by undergroundwires** [comment]
Closing this as there has been no response in a month.

**Comment by JKawmi** [comment]
I am having the same issue, should I open another issue/ticket?

> Can you copy & paste or attach the generated script file that you executed?

[privacy-script.zip](https://github.com/undergroundwires/privacy.sexy/files/7599226/privacy-script.zip)

> Which version of Windows are you running ? (Start menu -> enter About your PC -> Post what you see as "version" under "Windows Specifications")

I am running Windows 11 21H2 (Build 22000.348)

![image](https://user-images.githubusercontent.com/13506188/143326784-0437e7c3-676b-4a14-998e-fb8c58ca91b3.png)

**Comment by JKawmi** [comment]
Trying to open the language selector does not show anything and causes this Error in the Event Viewer

> Faulting application name: ShellExperienceHost.exe, version: 10.0.22000.132, time stamp: 0xdd210a66
> Faulting module name: Windows.UI.QuickActions.dll, version: 10.0.22000.348, time stamp: 0x6bfd3a05
> Exception code: 0x80000003
> Fault offset: 0x00000000000431d1
> Faulting process id: 0x1f14
> Faulting application start time: 0x01d7e18f1f43c518
> Faulting application path: C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy\ShellExperienceHost.exe
> Faulting module path: C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy\Windows.UI.QuickActions.dll
> Report Id: 6abb239b-0d70-4931-b112-1440c99cefa7
> Faulting package full name: Microsoft.Windows.ShellExperienceHost_10.0.22000.71_neutral_neutral_cw5n1h2txyewy
> Faulting package-relative application ID: App

![image](https://user-images.githubusercontent.com/13506188/143327339-742b2673-2fb1-49d8-a215-9df8e4e0a667.png)

I hope this helps!

**Comment by JKawmi** [comment]
Apparently the new action center was also not working, but I did not notice it yesterday. However, I got them to work again by running

> Dism /Online /Cleanup-Image /ScanHealth
> Dism /Online /Cleanup-Image /CheckHealth
> sfc /scannow

The first two did not find any corrupted components, but the third one found some corrupted files and repaired them
![image](https://user-images.githubusercontent.com/13506188/143452005-f9b7bb95-2ea8-44fd-9d23-665c3f81fc91.png)

and now the two menus are working again :)

**Comment by undergroundwires** [comment]
Thanks for the information @JKawmi, you provided with all necessary information to be able to reproduce this. I will take a look when I have time. Re-opening this 👍

**Comment by undergroundwires** [comment]
After some hardcore testing with many VMs this weekend I found out that I found that it was caused by "`Microsoft AAD Broker Plugin app`" script. Reverting it would solve the issue.

Btw could not reproduce in Windows 10 21H1 but could reproduce in Windows 11 21H2 with your provided script.

It seems like this app is doing more than we thought. This script has caused another problem before #54. In a future patch I'll document and unrcommend it, or consider removing it completely from privacy.sexy.

Thank you again for providing information to help diagnosing and resolving this issue.

**Comment by undergroundwires** [comment]
After some hardcore testing this weekend I found out that it was caused by "`Microsoft AAD Broker Plugin app`" script. Reverting it would solve the issue.

Btw could not reproduce in Windows 10 21H1 but could reproduce in Windows 11 21H2 with your provided script.

It seems like this app is doing more than we thought. This script has caused another problem before #54. In a future patch I'll document and unrcommend it, or consider removing it completely from privacy.sexy.

Thank you again for providing information to help diagnosing and resolving this issue.
